lua/orgmode/init.lua

```diff
@@ -59,6 +61,15 @@ function Org:setup_autocmds()
     group = org_augroup,
     callback = function()
       require('orgmode').reload(vim.fn.expand('<afile>:p'))
+      if config.org_startup_folded == 'overview' then
+        vim.opt_local.foldlevel = 0
+      elseif config.org_startup_folded == 'content' then
+        vim.opt_local.foldlevel = 1
+      elseif config.org_startup_folded == 'showeverything' then
+        vim.opt_local.foldlevel = 99
+      elseif config.org_startup_folded == 'inherit' then
+        vim.opt_local.foldlevel = vim.opt.foldlevel:get()
```

Comment:
I don't think we need this. It should inherit it automatically. We just need to not do anything in case of `inherit` value.

Reply:
Done as part of the refactor to place this in the config class. Take a look.